lib/env/roulette.py

```diff
@@ -27,7 +27,7 @@ def  step(self, action):
 			reward =  -1.0
 
 		self.budget = self.budget+reward
-		return  0, reward,  False, {}
+		return  0, reward,  False, {}, self.budget
```

Comment:
This works, but usually we put the information after each step in the variable that you see to the left of self.budget when you're returning the value. 

you wanna introduce a render method like the following:

```
def render(self, mode='human', close=False):
        # Render the environment to the screen
        profit = self.net_worth - INITIAL_ACCOUNT_BALANCE

        print(f'Step: {self.current_step}')
        print(f'Balance: {self.balance}')
        print(
            f'Shares held: {self.shares_held} (Total sold: {self.total_shares_sold})')
        print(
            f'Avg cost for held shares: {self.cost_basis} (Total sales value: {self.total_sales_value})')
        print(
            f'Net worth: {self.net_worth} (Max net worth: {self.max_net_worth})')
        print(f'Profit: {profit}')
```

Reply:
Of course, in your case your variables will be only budget. Try to think of a few more things that we would like to know throughout a game session!

Reply:
Just made some modifications and created a render method!